tests/integration/image-builders/release/Makefile

```
@@ -18,3 +21,22 @@ digest:
 push:
 	podman push quay.io/oc-mirror/release/test-image:v0.0.1 --authfile=$(AUTHFILE)
+
+sign: sign-cosign sign-tag-legacy
+.PHONY: sign
+
+sign-cosign:
+	COSIGN_PASSWORD="" cosign sign --key $(COSIGN_KEY) --signing-config=$(COSIGN_SIGNING_CONFIG) --tlog-upload=false quay.io/oc-mirror/release/test-image:v0.0.1
+	COSIGN_PASSWORD="" cosign sign --key $(COSIGN_KEY) --signing-config=$(COSIGN_SIGNING_CONFIG) --tlog-upload=false quay.io/oc-mirror/release/test-release-index:v0.0.1
+.PHONY: sign-cosign
+
+sign-tag-legacy:
```

checkmake (0.3.2)

[warning] Target body for "sign-tag-legacy" exceeds allowed length of 5 lines (8).

(maxbodylength)